### PR TITLE
Issue 137 - created the dashboard view and allow to be rendered

### LINKED ISSUE #137: [1] Create a course dashboard view and let the action render
Check `Course::DashboardsController`

### PR DESCRIPTION
**Pull request by jamiesonreinhard**
Closes #137

Updated Course::DashboardsController to stop the redirect to sections view. Created dashboards folder in views and added show.html.erb page.

### CODE REVIEW

**Comment by caioertai** [comment]
Nice.

File: app/views/course/dashboards/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1 class="mb-3">Hi, <%= current_user.email %></h1>
+<%= link_to "Learn", course_section_path(@section), class: "btn btn-primary" %>
```
Comment (caioertai) [inline app/views/course/dashboards/show.html.erb:2]: Cool. I like how you incorporated the link to the section. We're gonna need to change this to be a more dynamic on another PR, but that's a good start.

File: app/controllers/course/dashboards_controller.rb [hunk 2]
```diff
@@ -5,6 +5,5 @@ def show
     # TODO: make this not insane!
     @section = policy_scope(Chapter).first.sections.first
```
Comment (caioertai) [inline app/controllers/course/dashboards_controller.rb:7]: So. I'll try and setup some actions to help with this, so it makes the PR process easier. But right now my rubocop accuses this:
image

Also. Feel free to remove that comment. It's not insane anymore after your change. :grin:

File: app/views/course/dashboards/show.html.erb (hunk 1 above)
Comment (jamiesonreinhard) [reply app/views/course/dashboards/show.html.erb:2]: Definitely. I included it as a sort of placeholder for now!

File: app/controllers/course/dashboards_controller.rb (hunk 2 above)
Comment (jamiesonreinhard) [reply app/controllers/course/dashboards_controller.rb:7]: Sounds good. I will remove the comment and update the pull request.

**Review by caioertai — APPROVED** [review]
Cool!

**Comment by caioertai** [comment]
I added `Closes ##` to the pr to enable github magic when this gets merged, @jamiesonreinhard .

**Comment by becevi** [comment]
Reviewed. Nice Jamieson!

**Review by becevi — COMMENTED** [review]
Cool!
